app-admin/vuls/vuls-0.9.0-r1.ebuild

```diff
@@ -250,10 +253,6 @@ IUSE="policykit systemd"
 SLOT=0
 
```

Comment:
add `RESTRICT="mirror"` at this line please

Reply:
Added